test/components/views/rooms/__snapshots__/RoomHeader-test.tsx.snap

```diff
@@ -62,13 +62,6 @@ exports[`RoomHeader does not show the face pile for DMs 1`] = `
       >
         <div />
       </button>
-      <button
```

Comment:
Are the styles not required if the feature is enabled?

Reply:
I'm not sure I understand the question?

Reply:
It's before coffee :laughing: I did miss, that this is a snapshot.